src/app/components/category-products/category-products.component.html

```diff
@@ -8,24 +8,117 @@ <h5 class="offcanvas-title">Filter</h5>
             <button type="button" class="btn-close" aria-label="Close"></button>
           </div>
 
-          <div class="offcanvas-body ps-lg-2 pt-lg-0">
+          <div class="offcanvas-body ps-lg-2 pt-lg-0" [formGroup]="filterForm">
             <div class="mb-8">
               <h5 class="mb-3">Categories</h5>
               <ul class="nav nav-category">
                 <li *ngFor="let category of categories$ | async" class="nav-item border-bottom w-100 collapsed"><a
-                    class="nav-link">{{category.name}}<i class="feather-icon icon-chevron-right"></i></a></li>
+                    class="nav-link" [routerLink]="'/categories/'+category.id">{{category.name}}<i
+                      class="feather-icon icon-chevron-right"></i></a></li>
               </ul>
             </div>
-          </div>
+            <div class="mb-8">
+
+              <h5 class="mb-3">Stores</h5>
+              <div>
+                <input type="search" placeholder="Search by store" class="form-control mb-3" formControlName="stores">
+              </div>
 
+                <div *ngFor="let store of searchedStores$ | async" class="form-check mb-2"><input [id]="store.id"
+                    class="form-check-input" type="checkbox" (click)="onStoreChanged(store)" value="store.id" formControlName="store">
```

Comment:
you should use valueChanges, not click

Reply:
I try but I don.t know how use valueChanges on checkbox input, can we marge this branch and If i have time i try again

Reply:
ok